core/app/assets/stylesheets/glass/components/_icons.scss

```diff
@@ -198,232 +198,230 @@
   color: #fff;
 }
 
-body {
-  .icon:not(.icon-fe) {
```

Comment:
This `not(.icon-fe)` will be necessary in the future, but we can have a separate PR to pull that back in.